src/hello-n.py

```diff
@@ -1,6 +1,10 @@
 
 # this asks the user for a number and turns the answer
 # into an integer
-n = int(input("How many times should I loop? "))
+
 
 # Now, print "Hello, World!" n times.
+for x in range(5):
```

Comment:
I think the idea was that it should loop the number of times the user requested, not always five.